README.md

```diff
@@ -0,0 +1,30 @@
+phpspec
+=======
+
+The main website with documentation is at `http://www.phpspec.net <http://www.phpspec.net>`_.
```

Comment:
You can remove the backticks and just have the URL, in markdown it will automatically be converted to a link. So it will just be:

The main website with documentation is at http://www.phpspec.net